src/cli/datasets/__init__.py

```diff
@@ -12,7 +12,7 @@
 # permissions and limitations under the License.
 
 from ._main import datasets
-from .compute_catch22 import compute_catch22  # type: ignore
+# from .compute_catch22 import compute_catch22  # type: ignore
```

Comment:
Why not just remove this?

Reply:
This function rely on a package called catch22, but this package was renamed suddenly, and there was a problem with the installation, and the old version of the package could not be installed, so we comment it and will add it after solving the problem of this package.